go/registry/api/api.go

```diff
@@ -572,18 +572,16 @@ func VerifyRegisterNodeArgs( // nolint: gocyclo
 		)
 		return nil, nil, err
 	}
-	certPub, err := verifyNodeCertificate(logger, &n)
+	_, err := verifyNodeCertificate(logger, &n)
 	if err != nil {
 		return nil, nil, err
 	}
-	if !sigNode.MultiSigned.IsSignedBy(certPub) {
```

Comment:
Why remove this check, we should sign with and verify both signatures.

Reply:
This was also removed for debugging and is now back.